www/src/pages/guidelines/typography.js

```diff
@@ -0,0 +1,600 @@
+              Gatsby provides a modern framework for turning content into
+              feature-rich, visually engaging apps and websites.
+            </Heading>
+            <Text fontSize={{ xxs: 2, md: 3 }} mb={7}>
+              <p>
+                Over the last few years, the modern Javascript ecosystem has
+                created tools that allow developers to build quicker with fewer
+                bugs. Gatsby gives you easy access to features like modern
+                Javascript syntax, code bundling and hot reloading, without
+                having to maintain custom tooling. Build app-like experiences
+                faster — with Gatsby.
+              </p>
+              <p>
+                Websites come in a thousand different flavors. Timeframes,
+                budgets, interactivity requirements and content systems can vary
+                wildly from one project to the next.
+              </p>
+              <p>
+                This variety puts website teams between a rock and a hard place.
+                They often have to maintain frontends built in multiple
+                development systems, stretching their developers’ skill sets.
+                Implementing the same dropdown in five different frameworks can
+                be a huge headache. But what’s the alternative — turn down good
+                client projects?
+              </p>
+              <p>
+                To add to the difficulty, when your UI development framework is
+                coupled to your client’s CMS backend, it doesn’t just cause
+                technical problems; it causes people problems. It makes your
+                team’s staffing plans dependent on specific projects. It
+                hamstrings your ability to respond to changing client
+                requirements by shifting resources around.
+              </p>
+            </Text>
+            <Button bg="black">Read more</Button>
+          </Box>
+        </ContentColumn>
+      </Columns>
+    </Section>
+
+    <ThemeProvider theme={themeInverted}>
+      <Container
+        bg="white"
+        py={8}
+        my={8}
+        textAlign="center"
+        css={{ WebkitFontSmoothing: `antialiased` }}
+      >
+        <Text fontSize={1} color="blackFade.60">
+          Simple landing page (using an inverted theme)
+        </Text>
+
+        <Box maxWidth={1040} mx="auto">
+          <PageHeading
+            fontSize={{ md: 11 }}
+            lineHeight="solid"
+            maxWidth="48rem"
+            mb={3}
+            mx="auto"
+          >
+            Stop managing content.
+            <br /> Start telling your story.
+          </PageHeading>
+          <Intro color="blackFade.80" maxWidth="40rem" mx="auto" mb={6}>
+            Gatsby brings your content to the edge for lightning fast, safe
+            website delivery with no CMS overhead.
+          </Intro>
+          <Button
```

Comment:
Same as the `Simple landing page` text: it is difficult to read at the current size / color combination.

Reply:
Bumped both this and `Simple landing page" from `blackFade.60` to `blackFade.70`, which gets us to a contrast ratio of 7.37. I still have lots to do here that will help in preventing this from happening (one thing is to add aliases for safe text colors), but looking at the numbers this should be OK now.